bin/nomplate.js

```diff
@@ -61,7 +67,7 @@ function validate(args) {
   }
 }
 
-function execute(args) {
+function execute(args) {  
```

Comment:
Looks like you need to update your editor to automatically remove trailing whitespace.